resources/backend/js/gutenberg.js

```diff
@@ -159,6 +159,19 @@ function convertKitGutenbergRegisterBlock( block ) {
 											}
 										);
 									}
+
+									// Sort field's options alphabetically by label.
+									fieldOptions.sort(
```

Comment:
Why not use the [built-in comparison function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)?